tests/unit/DebugTest.php

```diff
@@ -49,4 +50,33 @@ function () {
             }
         );
     }
+
+    /**
+     * Tests function variable for object does not throw a exception
+     *
+     * @author limx <715557344@qq.com>
+     * @link   #13315
+     * @since  2018-03-08
+     */
+    public function testVariableForObjectNotThrowException()
+    {
+        $this->specify(
+            "Function variable for object not work expect",
+            function () {
+                try {
+                    (new Dump(null, true))->variable('xxx');
+                    (new Dump(null, true))->variable(111);
+                    (new Dump(null, true))->variable([1, 2, 3]);
+
+                    $obj = new \stdClass();
+                    $obj->test = 'value';
+                    (new Dump(null, true))->variable($obj);
+
+                    expect(true)->true();
+                } catch (\Exception $ex) {
+                    expect(false)->true();
+                }
```

Comment:
Here is not necessary to use try/catch because tests will fail in case of exception.